zipline/data/benchmarks.py

```diff
@@ -12,50 +12,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import pandas as pd
```

Comment:
Could you please create a separate PR for the GSPC - SPY related fixes?
It is an important fix, we shall not sneak in here :)